pkgs/development/libraries/librime/default.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   pname = "librime";
-  version = "1.5.0";
+  version = "1.5.2";
```

Comment:
1.5.3 was released recently, would you mind updating it? https://github.com/rime/librime/releases/tag/1.5.3

Reply:
Thanks for the review.
`librime` has been updated to the latest release.